src/components/Header.tsx

```diff
@@ -17,15 +26,21 @@ export default function Header() {
           </Link>
         </div>
         <div>
-          <Link href="/home">
-            <button className="btn btn-primary ml-3">Dashboard</button>
-          </Link>
-          <Link href="/create-event">
-            <button className="btn btn-primary ml-3">Create Event</button>
-          </Link>
-          <Link href="#FAQ">
-            <button className="btn btn-primary ml-3">FAQ</button>
-          </Link>
+          {currentPath !== '/home' && (
+            <Link href="/home">
+              <button className="btn btn-primary ml-3">Dashboard</button>
+            </Link>
+          )}
+          {currentPath !== '/create-event' && (
+            <Link href="/create-event">
+              <button className="btn btn-primary ml-3">Create Event</button>
+            </Link>
+          )}
+          {currentPath !== '/#FAQ' && (
```

Comment:
What's the intent of this change? Is that when they click the FAQ button it makes it disappear?  Since it is on the same page it may not make sense to do that.